test/core/test_loss.py

```diff
@@ -0,0 +1,300 @@
+import os
+import unittest
+
+from fastNLP.core.dataset import DataSet
+from fastNLP.core.metrics import SeqLabelEvaluator
+from fastNLP.core.field import TextField, LabelField
+from fastNLP.core.instance import Instance
+
+from fastNLP.core.optimizer import Optimizer
+from fastNLP.core.trainer import SeqLabelTrainer
+from fastNLP.models.sequence_modeling import SeqLabeling
+
+import fastNLP.core.loss as loss
+import math
+import torch as tc
+import pdb
+
+class TestLoss(unittest.TestCase):
+
+	def test_case_1(self):
+		#验证nllloss的原理
+
+		print (".----------------------------------")
+
+		loss_func = loss.Loss("nll")
+
+		#pdb.set_trace()
+
+		y = tc.Tensor(
+				[
+					[.3,.4,.3],
+					[.5,.3,.2],
+					[.3,.6,.1],
+				]
+			)
+
+		gy = tc.LongTensor(
+				[
+					0,
+					1,
+					2,
+				]
+			)
+
+
+		y = tc.log(y)
+		los = loss_func(y , gy)
+
+		r = -math.log(.3) - math.log(.3) - math.log(.1)
+		r /= 3
+		print ("loss = %f" % (los))
```

Comment:
Unit tests should use `self.assertXXX` to evaluate the correctness, like `self.assertEqual(los, r)` in this test.
If assertions fail to pass, unit tests find something wrong.

Reply:
Wonderful.